migration/20171218-rebuild-materialized-work-with-genre.sql

```diff
@@ -0,0 +1,121 @@
+-- The default for lanes.inherit_parent_restrictions has changed
+-- to true. Change the setting for all lanes not based on custom lists.
+-- For all lanes based on custom lists, a human has made a decision
+-- to set the value one way or the other.
+update lanes set inherit_parent_restrictions=true where _list_datasource_id is null and id not in (select l.id from lanes l join lanes_customlists);
```

Comment:
not a big deal but in the clause at the end you could select lane id from lanes_customlists and not need the join